components/modal/src/Modal.tsx

```diff
@@ -167,9 +160,6 @@ export const Modal = ({
 		Backdrop: { component: Backdrop, styles: backdropStyles, attributes: backdropAttributes },
 	} = overrideReconciler(defaultOverrides, tokenOverrides, brandOverrides, componentOverrides);
 
-	const modalRef = useRef(null);
-	const headingRef = useRef(null);
```

Comment:
Theses refs weren't being used anywhere and were causing warning to pop up in movemoney. Based on git history it seems they were to be used for focusing inside the modal but there is a library that seems to be doing that now.